### PR TITLE
Added TS forecasting and classification nb, along with processed dataset

### PR DESCRIPTION
**Pull request by suppathak**
Added TS forecasting notebook, classification notebook, and processed data 

Related #3

### CODE REVIEW

File: notebooks/growth_model_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,1053 @@
+{
```
Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>maybe we should reword it to: <code>... differentiate customers based on their growth potentials and identify the top most important customers that the business should be aware of.</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642049'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>we can remove this cell, as the function is already defined in the <code>helper_functions.ipynb</code> nb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642060'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>shouldn't we be calculating the slope just for the 6month prediction window (i.e. <code>2015-04 to 2015-09</code>) and exclude the validation months i.e remove <code>2015-01 to 2015-03</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642067'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='13'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>Let's remove this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642072'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='14'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>Update the description to remove the parts where we mention that we have 3 different types of services</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642078'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='16'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>reword to: <code>....we need to provide the customer account name as input parameters to the function</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642083'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='20'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>we can remove this cell, as the function is already defined in the <code>helper_functions.ipynb</code> nb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642087'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='21'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>reword to: <code>....we successfully analyzed different customer accounts based on their tenure and monthly revenue, resulting in the following accomplishments</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642093'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='26'/>

File: notebooks/growth_model_forecast.ipynb [hunk 2]
```diff
@@ -0,0 +1,1174 @@
+{
```
Comment (hemajv) [inline notebooks/growth_model_forecast.ipynb:1]: <p>Maybe add a note here to mention and describe the Kaggle dataset we are using </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642102'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

Comment (hemajv) [inline notebooks/growth_model_forecast.ipynb:1]: <p>Can you also add a brief <code>Conclusion</code> section to add a summary of the notebook? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1253642106'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='23'/>

File: notebooks/growth_model_classification.ipynb (hunk 1 above)
Comment (suppathak) [reply notebooks/growth_model_classification.ipynb:1]: Good catch. Thanks Hema for pointing this out. I have now changed it.

**Comment by suppathak** [comment]
Thanks @hemajv for your feedback. I have made the changes . Please have a look . Thanks.

File: notebooks/growth_model_classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,1025 @@
...
```
Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>update to: <code>.....to segment the clusters into three distinct groups</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1258641059'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='14'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>I think we should add a line to describe what this plot shows:</p><p><code>In the above plot, we try to visualize the 3 different cluster groups by plotting their slope values (along x-axis) and their median predicted value (along y-axis).</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1258641070'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='17'/>

Comment (hemajv) [inline notebooks/growth_model_classification.ipynb:1]: <p>Can you include a line to summarize this plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1258641074'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='20'/>

File: notebooks/growth_model_forecast.ipynb [hunk 4]
```diff
@@ -0,0 +1,1186 @@
...
```
Comment (hemajv) [inline notebooks/growth_model_forecast.ipynb:1]: <p>Let's update this to something like:</p><p><code>In this notebook, we will be developing a time series forecasting model i.e. ARIMA model (add hyperlink for ARIMA: https://en.wikipedia.org/wiki/Autoregressive_integrated_moving_average) to predict customer/user growth that enterprises can leverage to help improve their businesses. We will be using a publicly available monthly revenue dataset from Kaggle (add the link for dataset).</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1258641080'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

Comment (hemajv) [inline notebooks/growth_model_forecast.ipynb:1]: <p>remove: <code>We will be fetching data to obtain the monthly revenue for all accounts.</code></p><p> </p><p>maybe also add an additional line to describe the data: </p><p><code>The dataset consists of user/customer IDs, month and the revenue for each month.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1258641086'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

Comment (hemajv) [inline notebooks/growth_model_forecast.ipynb:1]: <p>remove duplicate <code>has</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/#comment-1258641091'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

Comment (hemajv) [reply notebooks/growth_model_forecast.ipynb:1]: <p>can you remove this line: <code>We will be fetching data to obtain the monthly revenue for all accounts.</code>?</p>

**Comment by hemajv** [comment]
@suppathak thanks for the changes!

/lgtm 🚀
